Codeinwp/visualizer
# Fix/470

### Summary


Changed the javascript rendering in favor of php side rendering for the pro charts lock. 

Updated the pop-up links.

Added small changes to keep the code DRY. 

### Will affect visual aspect of the product

NO



### Test instructions


Check that the pro charts lock notice is working as expected when a license is expired.

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/visualizer-pro/issues/470.
